### PR TITLE
D8 1202a

### PR DESCRIPTION
**Pull request by FloQuinn**
this goes with cyberteam d8-1202

### CODE REVIEW

**Comment by FloQuinn** [comment]
Checked in fix

File: modules/access_affinitygroup/access_affinitygroup.module [hunk 1]
```diff
@@ -1154,6 +1164,8 @@ function sendEmailCampaign($campaignId, $campaignActivityId, $ccListIds) {
   }
 }
 
-// @todo is this a safe include... should we use a different path specifier.
+// this include for constant contant wrapper html will become obsolete
```
Comment (a-pasquale) [inline modules/access_affinitygroup/access_affinitygroup.module:1167]: Is this still right or did is this remaining as the Community template?

File: modules/access_affinitygroup/src/Commands/AffinityGroupCommands.php [hunk 2]
```diff
@@ -177,4 +203,188 @@ public function showAffinityGroups(string $agName='', $options = ['uidonly' => f
...
+            // $enode->get('date')->value is in UTC 2023-02-13T19:33:00  (this was input as 2:33 pm NY time)
+
+            // get the custom view display eventinstance.email_summary
+            $view_builder = \Drupal::entityTypeManager()->getViewBuilder('eventinstance');
+            $renderArray = $view_builder->view($enode, 'rollup_list');
+            $display = \Drupal::service('renderer')->renderPlain($renderArray);
+
+            //$this->output()->writeln($display);
+
+             //TEMP
```
Comment (a-pasquale) [inline modules/access_affinitygroup/src/Commands/AffinityGroupCommands.php:326]: Cleanup

File: modules/access_affinitygroup/src/Commands/AffinityGroupCommands.php [hunk 3]
```diff
@@ -177,4 +203,188 @@ public function showAffinityGroups(string $agName='', $options = ['uidonly' => f
...
+    }
+    /**
+     * @command access_affinitygroup:showEventsS
+     *
+     * @aliases showEventsS
+     * @usage   access_affinitygroup:showEventsS
+     *
+     * Get all eventseries.
+     */
+    public function showEventsS()
```
Comment (a-pasquale) [inline modules/access_affinitygroup/src/Commands/AffinityGroupCommands.php:337]: Are you using this somewhere?

File: modules/access_news/access_news.module [hunk 4]
```diff
@@ -256,12 +285,258 @@ function affinityGroupBroadcast(EntityInterface $node, $newsType) {
...
+        $pubDate = $node->field_published_date[0]->view('alt_teaser');
+        $pubDateDisp = \Drupal::service('renderer')->renderPlain($pubDate);
+
+        // For body section, meant to be a snippet, this will return either a nicely
+        // truncated version of the body (the length is configured on site via
+        // /admin/structure/types/manage/access_news/display/alt_teaser )
+        // or, if the user has set the "custom summary", will return that instead.
+        // TODO:  The body is returned with <p>body</p> markup, but the summary is not.
+        // (need to distiguish because the news items with and without custom summary
+        //  will have different spacing in the digest.  Or do something with css)
```
Comment (a-pasquale) [inline modules/access_news/access_news.module:385]: I think this was dealt with with CSS? Remove the TODO?

**Comment by FloQuinn** [comment]
I'm using those commands as a dev tool a lot.  We're not using any of those
commands in production.

File: modules/access_news/access_news.module (hunk 4 above)
Comment (FloQuinn) [reply modules/access_news/access_news.module:385]: Sorry, yes, I did fix in css. might be good to keep the note that regular summary and custom summary have this difference., which certainly isn't obvious.

**Pull request merged**